meinberlin/apps/organisations/forms.py

```diff
@@ -24,6 +24,7 @@ class Meta:
         }
 
     def clean(self):
+        self._validate_unique = True
```

Comment:
@sabinammm That was hard to find. Here in mB we use this form in the django-admin to validate the groups. In the clean-method of django, the cleaned_data is returned as well, but first, it's validated that the fields are unique. https://github.com/django/django/blob/7c318a8bdd66f8c5241864c9970dddb525d0ca4c/django/forms/models.py#L432 I added that to our method overwriting the clean-method as well.

Reply:
:pray: